.github/workflows/api_tests.yaml

```diff
@@ -6,7 +6,7 @@ on:
   workflow_dispatch:
   pull_request:
     paths:
-      - '.github/workflows/api_tests.yaml'
+      - './github/workflows/api_tests.yaml'
```

Comment:
Not the proper path to the workflow file

Reply:
its ignored without backslash